server/src/actors/store.rs

```diff
@@ -140,7 +140,10 @@ impl Handler<StoreEvent> for StoreForwarder {
             .producer
             .send(record, 0)
             .then(|result| match result {
-                Ok(Ok(_)) => Ok(()),
+                Ok(Ok(_)) => {
+                    metric!(counter("processing.event.produced") += 1, "type" => "event");
```

Comment:
Isn't this kind-of redundant to `event.accepted`?

Reply:
The idea is to have an additional sanity check that we're actually producing something to Kafka, and not just proxy the events.

Reply:
AFAIK event.accepted is incremented in either case (processing or not).